mysql-test/suite/innodb/t/innodb-page_compression_none.test

```diff
@@ -0,0 +1,34 @@
+--source include/have_innodb.inc
+
+--echo #
+--echo # MDEV-30825 innodb_compression_algorithm=0 (none) increments Innodb_num_pages_page_compression_error
+--echo #
+
+SET @save_compression_algorithm=@@GLOBAL.innodb_compression_algorithm;
+SET GLOBAL innodb_compression_algorithm=0;
+SELECT VARIABLE_VALUE INTO @compress_errors FROM INFORMATION_SCHEMA.GLOBAL_STATUS WHERE VARIABLE_NAME = 'Innodb_num_pages_page_compression_error';
+
+
+CREATE TABLE t (c INT) page_compressed=1 page_compression_level=4 ENGINE=InnoDB;
+INSERT INTO t VALUES (1);
+SET GLOBAL innodb_checksum_algorithm=crc32;
+CREATE TABLE t1 (c INT) page_compressed=1 page_compression_level=4 ENGINE=InnoDB;
+INSERT INTO t1 VALUES (1);
+
+FLUSH TABLES t, t1 FOR EXPORT;
+UNLOCK TABLES;
+
+SELECT VARIABLE_VALUE - @compress_errors AS NUMBER_OF_ERRORS FROM INFORMATION_SCHEMA.GLOBAL_STATUS WHERE VARIABLE_NAME = 'Innodb_num_pages_page_compression_error';
+
+DROP TABLE t, t1;
+SET GLOBAL innodb_compression_algorithm=@save_compression_algorithm;
+
+# Seems to be needed reset INNODB_CHECKSUM_ALGORITHM to default
+let $shutdown_timeout = 0;
+--source include/restart_mysqld.inc
```

Comment:
Why do we even mess with `innodb_checksum_algorithm` in the test? You already added a `.combinations` file that covers both `strict_crc32` and `strict_full_crc32`.

Reply:
because of hopefully temporary blindness and not being able to see the obvious in the middle of the test. Sigh. Thanks Marko.